conf/base.config

```diff
@@ -8,6 +8,7 @@
  * on the PATH. Runs in `local` mode - all jobs will be
  * run on the logged in environment.
  */
+import nextflow.util.MemoryUnit
```

Comment:
Is this needed?
We have that in the `nextflow.config` file, but no import:
https://github.com/nf-core/sarek/blob/ca2a0b0e41128f86d24ab683074592ca26b81df8/nextflow.config#L112-L125

Reply:
Ah, let me check, perhaps it was added during debugging.

Reply:
Can you check my other comments in the conversation as well?

Reply:
I got an error when removing it:
```
ERROR ~ Unable to parse config file: '/lupus/proj/ngi2016004/private/johannes/src/sarek_dev/nextflow.config'

  Compile failed for sources FixedSetSources[name='/groovy/script/ScriptAFC21C7604A8062E1F8130921F0D5B8E']. Cause: org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
  /groovy/script/ScriptAFC21C7604A8062E1F8130921F0D5B8E: 53: unable to resolve class MemoryUnit
   @ line 53, column 30.
         memory = {check_resource((params.singleCPUMem as MemoryUnit) * 2 * task.attempt)}
                                  ^

  /groovy/script/ScriptAFC21C7604A8062E1F8130921F0D5B8E: 56: unable to resolve class MemoryUnit
   @ line 56, column 30.
         memory = {check_resource((params.singleCPUMem as MemoryUnit) * task.attempt * task.attempt)}
                                  ^

  2 errors


 -- Check '.nextflow.log' file for details
```
So I don't think that it works.

Reply:
Is it needed in `nextflow.config` then?